### DISCUSSION TITLE
How do I define a workflow that switches camera on and off on a timer?

### DISCUSSION THREAD

**Discussion by ejwm** [bonsai-rx/bonsai#1921] [Q&A]
Hi, I'm very new to Bonsai and can't work out how to get a workflow that records a video for 10 minutes, then waits 20 minutes, then records again (for a repeat of 3 times). 
Thanks for any help

**Comment by bruno-f-cruz** [comment]
A quick search returned these:
- #1363

- #870

- #1463

Would any of them work?
